routes/users.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+
+const { Package, Order } = require("../model/Package");
+const { User } = require("../model/User");
+const { verifyJWTToken } = require("../middlewares/verifyJWTtoken");
+
+const { isDuplicate } = require("../utils/isDuplicate");
+
+const router = express.Router();
+
+router.post("/:userId/bookmark", async (req, res, next) => {
+  try {
+    const userId = req.params.userId;
+    const { action } = req.body;
+
+    const existUser = await User.findById(userId).lean();
+    if (!existUser) {
+      return res.status(404).json({ error: "존재하지 않는 사용자입니다." });
+    }
+
+    const userBookmark_idList = existUser.bookmark;
+
+    const userWithBookmarks = await User.findById(userId)
+      .populate("bookmark")
+      .lean();
+    const bookmarkList = userWithBookmarks.bookmark;
```

Comment:
```suggestion
    const bookmarkInfos = await User.findById(userId)
      .populate("bookmark")
      .lean().bookmark;
```

userWithBookmarks 가 중간에 있으므로 이해를 더 빠르게 도와주지는 않는 것 같습니다. (개인적으로는 좀 더 헷갈렸던 편) 해서 한번에 작성하셔서 가져오는 것도 좋을 것 같습니다.

Reply:
그럴수도 있다는 생각이 들어서 한번에 변환하는걸로 수정해보았습니다!🤓